packages/s3-presigned-post/src/createPresignedPost.integ.spec.ts

```diff
@@ -0,0 +1,103 @@
+import { NoSuchKey, S3 } from "@aws-sdk/client-s3";
+const FormData = require("form-data");
+import { createReadStream, rmSync, writeFileSync } from "fs";
+import { join } from "path";
+
+import { createPresignedPost } from "./createPresignedPost";
+
+describe(createPresignedPost.name, () => {
+  const Bucket = "aws-sdk-js-integration-test-s3-presigned-post";
+  const Key = "temp-file.txt";
```

Comment:
Choose file name prefix based on what's being tested, and add a suffix to ensure there's no collision if tests are run in parallel?

In legacy integration tests, and also the new ones in S3, we use `Date.now()`